### ISSUE TITLE
Integrate Cantaloupe into code for curate prod, test and Arch

### ISSUE THREAD

**Issue by tmiles2** [emory-libraries/dlp-curate#321]
Linked to #297 
Linked to #674
- [x] Setup  Cantaloupe on Arch - DevOps
- [x] Setup Cantaloupe on Test - Dev Ops
- [x] Setup Cantaloupe on Prod - Dev Ops

**Comment by eporter23** [comment]
Re-opening because this work may not have been completed.

**Comment by eporter23** [comment]
Moving into 0.5 release because we don't know how long this may take to complete relative to other 0.4 items

**Comment by SolomonHD** [comment]
Progress: 
UCLA has updated their Cantaloupe delegate script and they've seemly designed it around fedora. Also they've added some sort of authorization logic. 
https://raw.githubusercontent.com/UCLALibrary/cantaloupe-delegate/master/lib/delegates.rb

I've built a new cantaloupe for arch and I've experimented with the config and I believe I have it about right but there's a problem, the generated url logic for the image files isn't correct.
For example curate is sending the following url to iiif when trying to access [this](https://curate-arch.library.emory.edu/concern/curate_generic_works/945cc2fr0x-cor?locale=en) entry:

https://iiif-cor-arch.library.emory.edu/cantaloupe/iiif/2/2675r7sqvc4-cor%2Ffiles%2F033b606a-0f40-4eab-bacc-73958f444668/full/90,/0/default.jpg?t=1575345637503

But of course the objects are not being stored in the s3 buckets like that. The url should look more like this:
https://iiif-cor-arch.library.emory.edu/cantaloupe/iiif/2/8f93a0be3eee62032cfc8df81072e1bc43c85975/full/90,/0/default.jpg
Mainly the identifier needs to be a sha1 of the object instead of whatever it currently is.
